tests/example_app/src/app.py

```diff
@@ -56,5 +61,38 @@ def reverse_string(
     return ReverseStringOutput(reversed_string=reversed_string)
 
 
+@app.on_poll()
+def on_poll(
+    params: OnPollParams, client: SOARClient, asset: Asset
+) -> Iterator[Union[Container, Artifact]]:
+    # Create container first for artifacts
+    yield Container(
+        name="Network Alerts",
+        description="Some network-related alerts",
+        severity="medium",
+    )
+
+    # Simulate collecting 2 network artifacts that will be put in the network alerts container
+    for i in range(2):
```

Comment:
You could avoid the `i + 1` math by changing the range here
```suggestion
    for i in range(1, 3):
```